ResXManager.Model/IResourceTableEntryRule.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace tomenglertde.ResXManager.Model
+{
+    using System.Collections.Generic;
+    using System.ComponentModel;
+
+    using JetBrains.Annotations;
+
+    /// <summary>A rule that is validated against a entry of the resource table.</summary>
+    /// <remarks>
+    /// This is used to implement the different rules that are used to check the proper translation.
+    /// </remarks>
+    internal interface IResourceTableEntryRule : IResourceTableEntryRuleConfig
+    {
+        /// <summary>
+        /// Check the rule by validating all the <paramref name="values"/>.
+        /// </summary>
+        /// <param name="values">The values to check</param>
+        /// <param name="message">
+        /// The human readable message that contains the reason for the failure.
+        /// In case the rule check was successful this parameter will contain <see langword="null" />.
+        /// </param>
+        /// <returns>
+        /// <see langword="true"/> in case the values passed the check; otherwise <see langword="false"/>
+        /// </returns>
+        bool CheckRule([NotNull][ItemNotNull] IEnumerable<string> values,
```

Comment:
I renamed the methods as requested, how ever I did leave the parameters as is for now. The reason why you didn't find any calls that actually made use of the list, is because I missed one call that was around for the StringFormat check.

Find the method that is now called `HasRulesMismatches` in the `ResourceTableEntry` class. This is the one place where the list is actually required. I declared the functions with lists as parameters, because the original function for StringFormat was declared like this as well and just like now in one instance it was required.